securedrop-protocol/src/journalist.rs

```diff
@@ -31,10 +31,14 @@ pub struct JournalistClient {
     /// Journalist's long-term fetching key pair
     fetching_key: Option<JournalistFetchKeyPair>,
     /// Journalist's long-term DH key pair
-    /// TODO: Remove
+    /// TODO: Remove? Not for use with encryption, although it
+    /// may be needed as "key of last resort" - to discuss
     dh_key: Option<JournalistDHKeyPair>,
     /// Generated ephemeral key pairs (for reuse)
-    ephemeral_keys: Vec<JournalistOneTimeKeyBundle>,
+    one_time_pubkeys: Vec<JournalistOneTimeKeyBundle>,
+    /// TODO: store complete key bundles (private and pubkey)
+    /// and use instead of dh_key for encryption
+    /// one_time_keystore: TODO
```

Comment:
it might be good to instead of having a field with `Vec<JournalistOneTimeKeyBundle>` to instead store `Vec<(JournalistOneTimeMessagePQKeyPair, JournalistOneTimeMessageClassicalKeyPair, JournalistOneTimeMetadataKeyPair)>`

Reply:
(or defining a type `JournalistOneTimeKeystore` for this as I see below makes sense too)